CHANGES.md

```diff
@@ -24,6 +24,67 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.16.0
+
+- educational materials
+  - major readme refactoring changes + filename changes for public release of RAI Dashboard
+  - add missing doc for pred_y_dataset and min_child_samples
+  - add npm and yarn installation instructions
+  - add romanlutz to CODEOWNERS of raiwidgets
+  - update NOTICE file
+  - add pull request template
+- new features
+  - ## Counterfactual
+    - counterfactual list style updates: highlight counterfactual examples columns with different color, and bold size
+    - highlight first and last row of counterfactual list
+    - implement list method for counterfactual manager
+  - ## Error Analysis
+    - add scrollbar area around matrix cells when there are many categories
+    - add warning message for downsampled explanation data in ErrorAnalysis
+- breaking changes
+- bug fixes and tests
+  - ## Responsible AI Dashboard
+    - add e2e UI tests for Dataset Explorer - whisker plot
+    - add e2e UI tests for Dataset Explorer - scatter plot
+    - add e2e UI tests for Dataset Explorer - flyout
+    - add e2e UI tests for Dataset Explorer - cohort scenarios
+    - add e2e UI tests for Model Statistics - chart tests
+    - refactor binding this with arrow func
+    - add error message for duplicate cohort name
+    - handle context cohort delete cases for data explorer
+  - ## RAIInsights
+    - throw error on mutating predict function in RAIInsights
+    - use spilt orientation to save train and test dataframe to address duplicate indices condition in them
+    - add .json extension to dtypes files and add tests
+    - add predict and predict_proba outputs to RAIInsights save method
+    - refactor RAIInsights load function into smaller functions
+    - use getattr to reference predict_proba
```

Comment:
I think this PR was edited to refactor certain logic for detecting if model is classifier. https://github.com/microsoft/responsible-ai-toolbox/pull/1075 (Refactor classifier detection logic into common function)

Reply:
updated!